packages/frontend/src/pages/public/CreateHousePage.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Card, Text, Modal, Input, Backdrop } from '@nextui-org/react';
+import React from 'react';
+import { useApi } from '../../hooks/useApi';
+
+interface CreateHousePageProps {}
+
+const CreateHousePage: React.FC<CreateHousePageProps> = () => {
+  const { data } = useApi('/api/v1/ping', {
+    method: 'get',
+  });
+  if (data) {
+    const { env, time } = data;
+    console.log({ env, time });
+  }
+
+  const [createVisible, setCreateVisible] = React.useState(false);
+  const createHandler = () => setCreateVisible(true);
+  const closeCreateHandler = () => {
+    setCreateVisible(false);
+  };
+
+  return (
+    <div className="h-screen overflow-y-auto bg-gradient-to-b from-land_page_bg_start to-land_page_bg_end">
+      <div>
+        <Modal open={true} animated={true} preventClose={true} width={'50%'}>
```

Comment:
Does this modal actually need to be its own route? Can it just be a component that's used in `ManageAccountPage`

Reply:
Agree. It is nice to refactor this modal out so that could potentially reuse.

Reply:
@FriedhelmWS  I agree with you. Initially, we put the modal inside the ManagementAccount Page. Cause in the issue, it mentioned that creating house be a page. So we move it as a page.  @uoa-aron527 will move this back.

Reply:
I have moved it back to ManageAccountPage and the other changes are also made.